assets/css/fonts.css

```diff
@@ -34,7 +34,7 @@
 }
 
 @font-face {
-    font-family: ExpensifyMono-Bold;
+    font-family: ExpensifyMono-Regular;
```

Comment:
So for whatever reason, it looks like the CSS is actually calling `ExpensifyMono-Bold` when we are using bold mono. So technically this is working fine now. However, we could update the font family name and then make updates elsewhere if we want to, it would be cleaner that way. 

<img width="944" alt="image" src="https://user-images.githubusercontent.com/2319350/206309395-2985b414-26d3-4b5f-ba5f-d94b36ecb4ef.png">

Reply:
Looks like we use a helper function to get the correct mono fonts, so I'll revert this one for now. 

https://github.com/Expensify/App/blob/master/src/styles/StyleUtils.js#L329-#L335